packages/date-parser/src/refiners/ambiguity.test.js

```diff
@@ -0,0 +1,16 @@
+import ambiguity from './ambiguity';
+
+describe('ambiguity refiner', () => {
+  it('can handle empty knowValues case', () => {
+    const inputResults = [
+      {
+        start: {
+          knownValues: {},
+        },
+      },
+    ];
+
+    const res = ambiguity.refine(null, inputResults);
+    expect(res).toEqual([]);
+  });
+});
```

Comment:
weird, there was previously no test covering this?  
If it's because Chrono has upgraded and won't return empty knownValues anymore, then we should just remove this refiner.

Reply:
Can you try checkout old codes and try removing this refiner to see if any test fails?

Reply:
Removed the refiner on old codes makes this test failed.

![image](https://user-images.githubusercontent.com/11194590/129658315-e56f4a55-b918-44bc-8909-60be534df9ab.png)

However, remove it on the new codes doesn't make any test failed. So I'll remove this refiner on the new code